.github/workflows/release-lambda-init-containers.yml

```diff
@@ -0,0 +1,36 @@
+name: Agent Release Lambda Layers and K8s Init Containers
+
+on:
+  workflow_run:
+    workflows: ["Create Release"]
+    types:
+      - completed
+
+jobs:
+  release-tags:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
+        with:
+          fetch-depth: 2
+      - uses: actions/setup-node@v4
+        with:
+          node-version: 'lts/*'
+      - run: npm install
+      - run: |
+          git config user.name ${GITHUB_ACTOR}
+          git config user.email gh-actions-${GITHUB_ACTOR}@github.com
+      - id: get_tag
+        run: echo "latest_tag=$(cat package.json | jq .version)" >> $GITHUB_OUTPUT
+      - name: Create release tags for Lambda and K8s Init Containers
+        run: |
+          RELEASE_TITLE="New Relic Node Agent v${{ steps.get_tag.outputs.latest_tag }}.0"
```

Comment:
looks like this may need to be updated based on the .net agent's teams changes [here](https://github.com/newrelic/newrelic-dotnet-agent/commit/74c0f8d9992cc9fe6d95558b6307ff96a0fdd7bf)